joomla/joomla-cms
# Set correct default values for user creation in the backend

### Summary of Changes

https://github.com/joomla/joomla-cms/blob/3.6.4/administrator/components/com_users/config.xml#L45-L68

Default values should be send mail to admin and admin activation.

That comes form here: https://github.com/joomla/joomla-cms/commit/cedd2d2e518b733c5fabfab45ce099709faccf69 and got reverted between that. This implements it back.
### Testing Instructions

install https://github.com/zero-24/joomla-cms/archive/install_SQL.zip
confirm the settings are equal to the XML
### Documentation Changes Required

None.
